r-notebook/Dockerfile

```diff
@@ -46,3 +46,6 @@ RUN conda install --quiet --yes \
     'r-hexbin=1.27*' && \
     conda clean -tipsy && \
     fix-permissions $CONDA_DIR
+
+# Fix for devtools https://github.com/conda-forge/r-devtools-feedstock/issues/4
+RUN echo "TAR=/bin/tar" > $HOME/.Renviron
```

Comment:
Users of these images often mount a host fodler as the home directory. In that case, the `.Renviron` file will be missing. Is there a another location R looks for configuration  (e.g., somewhere in `/etc`) where we can put this file instead?